llvm/test/CodeGen/PowerPC/aix-spills-for-eh.ll

```diff
@@ -0,0 +1,300 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mcpu=pwr9 -mattr=+altivec -verify-machineinstrs --vec-extabi \
+; RUN:   -mtriple=powerpc-unknown-aix < %s  | FileCheck %s --check-prefix 32BIT
```

Comment:
nit:
Please add `-ppc-asm-full-reg-names -ppc-vsr-nums-as-vr` to the run lines. It makes the test easier to read.

Reply:
done